### DISCUSSION TITLE
Ente Auth - Mac menu bar pop-up, instead of separate window application

### DISCUSSION THREAD

**Discussion by paragpindoria** [ente-io/ente#4501] [Enhancements]
App should pop-up in the menu bar so its easily accessible when it is needed. It clears clutter as you don't need another window open consistently.
Example mockup:
<img width="701" alt="image" src="https://github.com/user-attachments/assets/e73892b8-fa6e-4dac-82ab-0ac48c973961" />

**Comment by karlacio** [comment]
I was about to ask for the same feature and found this request.
I absolutely support this request!
